tests/TestHelpers/Fuzz.elm

```diff
@@ -0,0 +1,130 @@
+module TestHelpers.Fuzz exposing
```

Comment:
i'm curious, where the name fuzz comes from? i've seen "factory" "mocker" "faker" etc, but never fuzz 🤔

maybe i didn't catch the ideia but this is the same concept as those other names right?

Reply:
I have two answers for you:

Why I chose this name:
Basically to follow the names from the [test package](https://package.elm-lang.org/packages/elm-explorations/test/latest/). You can see there's `TestHelpers.Random`, `TestHelpers.Shrink` and `TestHelpers.Fuzz`, which then are imported with the aliases `Random`, `Shrink` and `Fuzz` respectively, which are the same names you would use if you were to use the test package directly. These modules just provide `Generator`s, `Shrinker`s and `Fuzzer`s that are specific to our project.

Why it's named like this on the test package:
Probably because of [Fuzzing](https://en.wikipedia.org/wiki/Fuzzing) (or fuzz tests), as described in the [Test.fuzz documentation](https://package.elm-lang.org/packages/elm-explorations/test/latest/Test#fuzz), which also mentions other ways this kind of test is called.

(By the way, there's a pretty interesting text about shrinkers [here](https://package.elm-lang.org/packages/elm-explorations/test/latest/Shrink#what-are-shrinkers-and-why-do-we-need-them-), explaining what they are and how they can help us)